### ISSUE TITLE
LogManager.ForContext threw IOException

### ISSUE THREAD

**Issue by jcansdale** [github/VisualStudio#1477]
- GitHub Extension for Visual Studio version: 2.4.2
- Visual Studio version: 2017

__What happened__ (with steps, logs and screenshots, if possible)

1. Launched VS 2017 from VS 2015 under debugger
2. While loading the following exception occured

![image](https://user-images.githubusercontent.com/11719160/35990590-659c5a7e-0cfc-11e8-9334-1471a1e17bf8.png)

Maybe this is only showing up under the debugger and Serilog is handling it correctly.
Should we be splitting out logs per VS instance?

**Comment by grokys** [comment]
Yeah, I think we should probably be splitting the logs as you say.
